libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/model/RouteLineColorResources.kt

```diff
@@ -95,7 +99,10 @@ class RouteLineColorResources private constructor(
             "alternativeRouteModerateColor=$alternativeRouteModerateColor" +
             "alternativeRouteHeavyColor=$alternativeRouteHeavyColor" +
             "alternativeRouteSevereColor=$alternativeRouteSevereColor" +
-            "alternativeRouteCasingColor=$alternativeRouteCasingColor"
+            "alternativeRouteCasingColor=$alternativeRouteCasingColor" +
+            "routeClosureColor=$routeClosureColor" +
+            "alternativeRouteClosureColor=$alternativeRouteClosureColor" +
```

Comment:
This also applies to the rest of the fields.

```suggestion
            "alternativeRouteCasingColor=$alternativeRouteCasingColor, " +
            "routeClosureColor=$routeClosureColor, " +
            "alternativeRouteClosureColor=$alternativeRouteClosureColor, " +
```

Reply:
done